<launch>
    <rosparam file="$(find rp_semantic)/config/params_config.yaml" command="load"/>

    <!-- node name="semanticRGB_server" pkg="rp_semantic" type="semanticRGB_server.py"/-->
    <node name="fusion_node" pkg="rp_semantic" type="fusion_node" output="screen"/>
    <!-- node name="rviz" pkg="rviz" type="rviz"/-->
</launch>
